.github/workflows/module-requests.yml

```diff
@@ -0,0 +1,154 @@
+        required: false
+        default: "triage-all"
+        type: choice
+        options:
+          - analyze-single
+          - triage-all
+          - dry-run
+      issue_number:
+        description: "Issue number for analyze-single mode"
+        required: false
+        type: string
+
+permissions:
+  issues: write
+  contents: read
+  projects: write
+
+jobs:
+  module-request-handler:
+    runs-on: ubuntu-latest
+
+    # Only run for module requests or scheduled/manual triggers
+    if: |
+      github.event_name == 'schedule' ||
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'issue_comment' &&
+       contains(github.event.issue.labels.*.name, 'module: new') &&
+       contains(github.event.comment.body, '@claude analyze-module')) ||
+      (github.event_name == 'issues' &&
+       github.event.action == 'opened' &&
+       contains(github.event.issue.labels.*.name, 'module: new'))
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Verify Module Request Context
+        id: verify
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { eventName, payload } = context;
+
+            // Always proceed for scheduled and manual runs
+            if (eventName === 'schedule' || eventName === 'workflow_dispatch') {
+              return { isModuleRequest: true };
+            }
+
+            // For issue events, check for module request
+            const issue = payload.issue;
+            if (!issue) return { isModuleRequest: false };
+
+            const hasModuleLabel = issue.labels?.some(l => l.name === 'module: new');
+
+            if (hasModuleLabel) {
+              return { isModuleRequest: true, issueNumber: issue.number };
+            }
+
+            // Auto-add label if this looks like a module request
+            if (payload.action === 'opened' && issue.body?.includes('Name of the tool')) {
+              await github.rest.issues.addLabels({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: issue.number,
+                labels: ['module: new']
+              });
+              return { isModuleRequest: true, issueNumber: issue.number };
+            }
+
+            return { isModuleRequest: false };
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Exit if Not Module Request
+        if: fromJSON(steps.verify.outputs.result).isModuleRequest == false
+        run: |
+          echo "❌ Not a module request - exiting workflow"
+          exit 0
+
+      - name: Determine Operation Mode
+        id: mode
+        if: fromJSON(steps.verify.outputs.result).isModuleRequest == true
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { eventName, payload } = context;
+            const inputs = context.payload.inputs || {};
```

Comment:
I don't understand how this would work with the cron trigger.. That needs to loop over all issues, right?